dev/release/verify-release-candidate.sh

```diff
@@ -499,6 +499,35 @@ maybe_setup_conda() {
   fi
 }
 
+install_maven() {
+  MAVEN_VERSION=3.8.7
+  SYSTEM_MAVEN_VERSION=$(mvn -v | head -n 1 | awk '{print $3}')
```

Comment:
Do we need to handle the case where there is no existing Maven installation?
```suggestion
  if command -v mvn > /dev/null; then
    SYSTEM_MAVEN_VERSION=$(mvn -v | head -n 1 | awk '{print $3}')
    show_info "Found Maven version ${SYSTEM_MAVEN_VERSION} at $(command -v mvn)."
  else
    SYSTEM_MAVEN_VERSION=0.0.0
    show_info "Maven installation not found."
  fi
```

Reply:
Yes, we need this. Good catch!